docs/content/latest/yugabyte-cloud/release-notes.md

```diff
@@ -162,7 +162,7 @@ The following **AWS regions** are available:
 - **Clusters** - No support for scaling vCPUs on single node clusters.
 - **Metrics** - The **View Full Screen** option in charts on the cluster **Overview** and **Performance Metrics** pages does not work in some versions of Safari 14.0 (Big Sur).
 - **Metrics** - Some clusters in European regions may show occasional spikes in the YSQL Operations/sec chart. This is due to cluster health checks and can be ignored.
-- **Pausing clusters** - The rate shown in the **Pause Cluster** dialog does not include backup storage costs.
+- **Pausing clusters** - The Once Paused rates shown in the **Pause Cluster** and **Resume Cluster** dialog incorrectly refer to usage costs. The Once Resumed rate shown in the **Resume Cluster** dialog does not include additional disk storage costs.
```

Comment:
What info is incorrect is indicated correctly in this PR.  However, where it is incorrect is a little ambiguous.  Adding explanation below. Please see if it needs updates.  I

The 'Pause Cluster' dialog has Current Cost (refers to active cluster) and Once Paused (refers to paused cluster).
The 'Resume Cluster' dialog has - Current Cost (refers to paused cluster) and Once Resumed (refers to active cluster). 
Thus, 'incorrectly refers to usage costs' is in 2 places: (1) in 'Once Paused' rates in Pause cluster dialog and (2) in 'Current Cost' in Resume cluster dialog.  
And, 'does not include additional disk storage costs' is in 2 places: (1) in 'Current Cost' in Pause cluster dialog and (2) in 'Once Resumed' in Resume cluster dialog.